src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -223,118 +228,114 @@ export default class VictoryPromiseDisplay extends React.Component {
 
     if (this.state.defaultVPMode) {
       content = (
-        <section>
-          <Grid container>
-            <Grid item xs={12}>
-              <Card>
-                <CardContent>
-                  <Grid container direction="row" alignItems="center">
-                    <Grid item xs={7}>
-                      <CardHeader title="Victory Promise" />
-                    </Grid>
-                    <Grid item xs={5} align="right">
-                      <Button
-                        variant="contained"
-                        style={style.button}
-                        onClick={() => this.changeMode()}
-                      >
-                        Change Style
-                      </Button>
-                    </Grid>
-                  </Grid>
-                  <Grid className="victory-promise-layout" item container>
-                    <Grid item xs={12} container justify="center">
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Christian Character</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_CC}
-                      </HoverText>
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Intellectual Maturity</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_IM}
-                      </HoverText>
-                    </Grid>
-                    <Grid container justify="center">
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Lives of Service</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_LS}
-                      </HoverText>
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Leadership Worldwide</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_LW}
-                      </HoverText>
-                    </Grid>
-                  </Grid>
-                </CardContent>
-              </Card>
-            </Grid>
+        <Grid item xs={12} container justify="center">
+          <Grid>
+            <HoverText
```

Comment:
Do you notice a difference when leaving the Grid tag blank versus declaring it an item?

Reply:
Yes, Isaac said that if there are other Grids inside a Grid, I have to set the outer Grid as an item and container